src/Command/DataFlow/DataFlowTimeoutJobsCommand.php

```diff
@@ -26,18 +27,27 @@ class DataFlowTimeoutJobsCommand extends Command
     private $dataFlowJobRepository;
     private $jobTimeoutThreshold;
 
-    public function __construct(EventDispatcherInterface $eventDispatcher, DataFlowJobRepository $dataFlowJobRepository, $jobTimeoutThreshold)
+    public function __construct(EventDispatcherInterface $eventDispatcher, DataFlowJobRepository $dataFlowJobRepository)
     {
         parent::__construct();
         $this->eventDispatcher = $eventDispatcher;
         $this->dataFlowJobRepository = $dataFlowJobRepository;
-        $this->jobTimeoutThreshold = $jobTimeoutThreshold;
+    }
+
+    public function configure()
+    {
+        $this->addOption('timeout-threshold', 'tt', InputOption::VALUE_OPTIONAL, 'In minutes how long to wait before a non-complete job should timeout', 30);
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
     {
+        $timeoutThreshold = $input->getOption('timeout-threshold');
+        if (!\is_int((int) $timeoutThreshold)) {
+            throw new \InvalidArgumentException('timeout-threshold can only be an integer value.');
+        }
```

Comment:
`is_int((int) $timeoutThreshold` will always be true since all possible return values from [`InputInterface::getOption`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Input/InputInterface.php#L120) can be successfully cast as `int`.
Something like `(int) $timeoutThreshold == $timeoutThreshold && $timeoutThreshold > 0` should work (lifted from numerous discussions about this interesting subject on the Internets).